spot-ml/src/main/scala/org/apache/spot/SuspiciousConnectsArgumentParser.scala

```diff
@@ -8,25 +8,17 @@ object SuspiciousConnectsArgumentParser {
 
   case class SuspiciousConnectsConfig(analysis: String = "",
                                       inputPath: String = "",
-                                      scoresFile: String = "",
+                                      feedbackFile: String = "",
                                       duplicationFactor: Int = 1,
-                                      modelFile: String = "",
-                                      topicDocumentFile: String = "",
-                                      topicWordFile: String = "",
-                                      mpiPreparationCmd: String = "",
-                                      mpiCmd: String = "",
-                                      mpiProcessCount: String = "",
                                       topicCount: Int = 20,
-                                      localPath: String = "",
-                                      localUser: String = "",
-                                      ldaPath: String = "",
-                                      nodes: String = "",
                                       hdfsScoredConnect: String = "",
-                                      hdfsModelFile: String = "",
                                       threshold: Double = 1.0d,
                                       maxResults: Int = -1,
                                       outputDelimiter: String = "\t",
-                                      ldaPRGSeed: Option[Long] = None)
+                                      ldaPRGSeed: Option[Long] = None,
```

Comment:
the intention was to use it for debugging purposes,  so that's why it's not in the ml_ops.sh script... what happens in current code is that if it's set to -1 it doesn't "set" the seed which means that in LDA, a "random" number from the utils library is used to seed LDA